xls/public/c_api.h

```diff
@@ -199,6 +213,21 @@ bool xls_bits_eq(const struct xls_bits* a, const struct xls_bits* b);
 // returns 1, `xls_bits_get_bit(bits, 1)` returns 0.
 bool xls_bits_get_bit(const struct xls_bits* bits, int64_t index);
 
+// Returns the bytes in the given bits value. The caller must
+// free the returned `bytes_out` pointer via `free`,  The caller
```

Comment:
Shouldn't it be freed via `xls_bytes_free`?

Reply:
+1